metrics-service-package/src/main/resources/bin/start.sh

```diff
@@ -70,7 +70,7 @@ else
   nonStrictVerifySslCertificatesOfServices=true
 fi
 
-if [ `uname` = "OS/390" ]
+if [ "$(uname)" = "OS/390" ]
```

Comment:
does $() execute within ""? Not sure but I see ```if [ `uname` = "OS/390" ]; then``` everywhere else in the codebase working successfully.

Reply:
it executes on our system